textsubs.lic

```diff
@@ -890,4 +890,302 @@ TextSubs.add('Your skill in (.*) is at a zenith of enlightenment\.', 'Your skill
 # Commas when dealing with coins
 TextSubs.add('\d{1,3}(?=(\d{3})+(?!\d).*(Kronar|Dokora|Lirum))', '\&,')
 
+# Scroll subs to add guild name
+TextSubs.add('"Strange Arrow."', '"Strange Arrow(Analagous)."')
```

Comment:
Oops, I misspelled it.

The bacon the naga thing was just a personal joke for myself.  It shouldn't be in the file that's pushed out to everyone.

As far as changing it to :
scroll_label = '^(It is labeled)'
TextSubs.add('#{scroll_label} "Burden."', '\1 "Burden" (Analogous Patterns).')

What does the \1 do in that line?  I think I understand the scroll_label part and #scroll_label.